blocks/identity-block/themes/news.json

```diff
@@ -239,22 +239,24 @@
 	"login-form": {
 		"styles": {
 			"default": {
-				"font-family": "var(--font-family-primary)",
+				"font-family": "Inter",
+				"padding-bottom": "var(--global-spacing-5)",
```

Comment:
We need to use CSS logical properties in our default styles to not disrupt RTL clients. Could you please look over https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values and convert the physical properties in this file to their logical equivalents? Thank you.